openrarity/providers/opensea/client.py

```diff
@@ -16,20 +16,43 @@
 
 from .types import TokenAsset
 
-logging.basicConfig(stream=sys.stderr, level=logging.INFO)
+FORMAT = f"%(message)s"
+logging.basicConfig(format=FORMAT,stream=sys.stderr, level=logging.INFO)
```

Comment:
Just pass the value of `FORMAT` to the logging config. No need for a variable if it isn't reused later.